garrysmod/gamemodes/sandbox/entities/entities/gmod_winch_controller.lua

```diff
@@ -88,7 +86,7 @@ function ENT:Think()
 
 		if ( per == 0 ) then return true end
 
-		local spos = ( math.sin( ( self.ctime * math.pi * per ) ) + 1 ) * ( amp / 2 )
+		local spos = ( math.sin(  self.ctime * math.pi * per  ) + 1 ) * ( amp / 2 )

```

Comment:
```suggestion
		local spos = ( math.sin( self.ctime * math.pi * per ) + 1 ) * ( amp / 2 )

```
Remove the double spaces